### PR TITLE
Issue 47 - general skinning

### PR DESCRIPTION
**Pull request by cohen-j**
At the time of this draft, I have added some small changes such as padding (`.bg-basic, div.container, form {` in gloss.css) and a different hover effect for the navbar (in the gog-header custom element in layout.js).

image

I have also added the bottom border from the navbar to the main form of each page.

image

For upcoming changes, I may look at changing the look and spacing of some of the buttons.

### CODE REVIEW

**Comment by cohen-j** [comment]
Blurred the background a little more (maybe too much.. not sure?) so the white boxes don't blend in so much. Also, modified the paragraph saying "NB: A Gloss is an important thing..." to stand out more. 

image

Made sure the additional padding from my last commit didn't make glosses.html look strange by adding some margins to it.
image

Also changed the color of these buttons. 
image
image

**Comment by AbhilashKotha** [comment]
@cohen-j The mentioned changes work perfectly fine for me in my testing. I am no UX expert but my suggestion is to have a consistent margin instead of a step like one (marked in red boxes)

Screenshot 2023-10-09 at 12 46 11 AM

File: css/gloss.css [hunk 1]
```diff
@@ -471,4 +474,20 @@ line.selected.just {
 
 .addWitnessBtn{
   margin-bottom: 1em;
+}
+
+form#ngForm
```
Comment (cubap) [inline css/gloss.css:479]: Using IDs in a general CSS file should only be done if it really needs to be specific to that one element only.

**Review by cubap — COMMENTED** [review]
Looking good

File: css/gloss.css (hunk 1 above)
Comment (thehabes) [reply css/gloss.css:479]: As Patrick said this class will only apply to this one specific form.  If you want this to happen on other forms be sure to make it more generic.

**Review by thehabes — COMMENTED** [review]
Nice updates!  See the comments from Abhilash, as I think that is a good note.  Straight margins/borders/paddings so it doesn't have the step effect.

**Comment by cohen-j** [comment]
@thehabes @AbhilashKotha The alignment problem should be fixed now. 

image

And in regards to using form#ngForm in gloss.css instead of generalization - the element that it changes on glosses.html does not seem to be included in nor modified by the overall 'form' like other pages have. Do you have any suggestions on how I could work around this?

**Comment by AbhilashKotha** [comment]
great work @cohen-j, Can you also fix the alignment on the home page? 

image

**Comment by cohen-j** [comment]
@AbhilashKotha 

Should be good now.

image

**Review by AbhilashKotha — APPROVED** [review]

**Review by thehabes — APPROVED** [review]

**Pull request merged**